### ISSUE TITLE
Highlight CAD parts while creating issues and comments

### ISSUE THREAD

**Issue by ghackenberg** [ghackenberg/productboard#3]
When referencing CAD parts while creating a new issue or comment, these CAD parts are not highlighted yet. Currently, the parts are highlighted only after the issue or comment has been submitted. In the future, the CAD parts should be highlighted while editing the issue or comment making the system more user friendly.
